src/main/xar-resources/data/repo/repo.xml

```diff
@@ -92,6 +96,10 @@
         </variablelist>
         <para>For library packages <literal>repo.xml</literal> is optional. However, we recommend to
             always provide it for better tool integration.</para>
+        <para>The following is not mentioned in the EXPath spec, but if an image resource with a
+            name that begins with the string <literal>icon</literal> is found in the root directory
+            of an app, the eXist-db dashboard will display that user-supplied image instead of the
+            default eXist-db blue dotted <literal>X</literal>.</para>
```

Comment:
@djbpitt These are the lines containing the new contribution, correct? I just want to be sure I didn't miss anything in the other changes, which upon first glance appear to be whitespace-only.

Reply:
@joewiz Yes. The whitespace changes are a side-effect of pretty-printing in <oXygen/>. Should I have known of an easy way to exclude them from the commit? They certainly do make it harder to understand.

Reply:
@djbpitt No worries. To avoid this on repositories that don't have a coordinated whitespace policies, I apply format & indent to the element(s) I've edited instead of to the whole document. Right click on an element and from the contextual menu select Source > Format and Indent Element. Or from the menu bad select Document > Source > Format and Indent Element.

Reply:
@joewiz Got it. Thank you. Another day, another <oXygen/> feature I hadn’t known about …

Reply:
see https://github.com/eXist-db/documentation/pull/637 running `mvn clean package` or `mvn test` will enforce a unified linting of all xml files.